css/webkit.css

```diff
@@ -6,136 +6,16 @@ body
 	font-family:Nunito, sans-serif;
 	font-style:normal;
 	font-weight:600;
-  
 }
 
-
-.card {
-	  width  : 350px;
-	  height : 400px;
-	  box-shadow: var(--bbj-shadow-l);
-	  margin-right: 20px;
-	  margin-bottom: 20px;
-	  display: flex;
-	   
-}
-.cardpanel {
-	  padding-bottom: 10px;
-	
-}
-
-.cardtext {
-	padding-left:15px;
-	padding-right:15px;
-	padding-bottom:10px;
-}
-
-.cardimg{
-     width:350px;
-     height:235px;
-     background-size:contain;
-     background-repeat:no-repeat;
-     background-position-x:center;
-     background-position-y:center;
-}
-
-
-.gridholder{
-
-        width: calc( 100vw - 150px );
-        height: calc( 100vh - 250px );
-
-}
-
-.tileslist{
-
-        width: calc( 100vw - 140px );
-        height: calc( 100vh - 250px );
-}
-
-.tileslist .BBjPanel {
-        width: calc( 100vw - 150px );
-        height: calc( 100vh - 250px );
+.gridholder {
+	width: calc( 100vw - 150px );
+	height: calc( 100vh - 250px );
```

Comment:
I don't see any reference of this class anywhere in the code. As long as we are cleaning stuff, shouldn't we just remove things that are not being used?